src/main/webapp/app/controllers/submission/adminSubmissionViewController.js

```diff
@@ -590,14 +590,16 @@ vireo.controller("AdminSubmissionViewController", function ($anchorScroll, $cont
             "hasPrimaryDocument": $scope.hasPrimaryDocument
         };
 
+        var getLastActionLog = function() {
+            return $filter('orderBy')($scope.submission.actionLogs, 'actionDate', true)[0];
```

Comment:
I hope that this orderBy filter always returns an array with at least a size of one.
Otherwise this may cause errors.

Reply:
I was operating under every submission must have one action log. I will put in some conditions to make this safer.

Reply:
Updated.